runtime-modules/content/src/lib.rs

```diff
@@ -2225,6 +2225,7 @@ decl_module! {
         pub fn accept_incoming_offer(
             origin,
             video_id: T::VideoId,
+            witness_price: Option<<T as balances::Config>::Balance>
```

Comment:
in `buy_nft` I have used the term `price_commit` with the same semantics. If you think that `witness_price` is better please change that one also, so there is no confusion (since both params serve the same purpose)